resources/server/api/ogc/templates/wfs3/describeCollection.html

```diff
@@ -4,7 +4,7 @@
 <div class="row">
     <div class="col-md-6">
         <h1><a title="View items of: {{ title }}" href="{% for link in links %}
-        {% if link.type == "text/html" and link.rel == "items" %}
+        {% if link.type == &quot;text&sol;html&quot; and link.rel == &quot;items&quot; %}
```

Comment:
I don't think these changes are correct -- were they deliberate?

Reply:
Yes, `{{ static( ` and ` ) }}` become strings but the parts between does not. I assume my fix is wrong, but I don't know it should be written instead